src/MMALSharp.FFmpeg/Handlers/VLCCaptureHandler.cs

```diff
@@ -0,0 +1,33 @@
+﻿// <copyright file="VLCCaptureHandler.cs" company="Techyian">
+// Copyright (c) Ian Auty. All rights reserved.
+// Licensed under the MIT License. Please see LICENSE.txt for License info.
+// </copyright>
+
+namespace MMALSharp.Handlers
+{
+    /// <summary>
+    /// Helper-methods for invoking ExternalProcessCaptureHandler to output to VLC.
+    /// </summary>
+    public static class VLCCaptureHandler
+    {
+        /// <summary>
+        /// Listens for a request on the given port and begins streaming MJPEG images when a client connects.
+        /// </summary>
+        /// <param name="listenPort">The port to listen on. Defaults to 8554.</param>
+        /// <param name="echoOutput">Whether to echo stdout and stderr to the console or suppress it. Defaults to true.</param>
+        /// <returns>An initialized instance of <see cref="ExternalProcessCaptureHandler"/></returns>
+        public static ExternalProcessCaptureHandler StreamMJPEG(int listenPort = 8554, bool echoOutput = true)
+        {
+            var opts = new ExternalProcessCaptureHandlerOptions
+            {
+                Filename = "cvlc",
+                Arguments = $"stream:///dev/stdin --sout \"#transcode{{vcodec=mjpg,vb=2500,fps=20,acodec=none}}:standard{{access=http{{mime=multipart/x-mixed-replace;boundary=7b3cc56e5f51db803f790dad720ed50a}},mux=mpjpeg,dst=:{listenPort}/}}\" :demux=h264",
```

Comment:
A few comments about this:

1) I can see the bitrate and fps values are static here, do we need to have a way of retrieving them from this class? It could be that the `MMALCameraConfig` class is moved into `MMALSharp.Common`, it might be a bit cleaner than adding additional parameters to the method?
2) Can you provide any additional context around the "boundary" argument? I have looked online and can see this same GUID referenced in many places but no context as to what this GUID represents.

Reply:
1. That bitrate and fps caps the MJPEG output stream, it isn't related to the input/camera settings. If you have a busy wifi, you could set those lower to reduce congestion. The browser will try to pull MJPEG frames back as fast as possible, it's up to the server how quickly a new frame is available to send. So probably arguments to the call make more sense.

2. The value to the boundary argument is arbitrary and is part of the MIME-type declaration in the response headers. VLC just happens to hard-code that particular string. I have wondered why, so I did some searching, but came up empty-handed.

Boundaries are discussed in this SO post and reply: https://stackoverflow.com/questions/47729941/mjpeg-over-http-specification

There's a small bug though, that GUID should be prefixed by two dashes. The answer on that SO post explains the two dashes are part of the MIME RFC, so I'll add them back. I saw the boundary marker referenced online both with and without dashes, and it works without them, so I left them off, figuring less is more. I've also seen people use varying longer strings of dashes, so I suppose it's not well understood. Anyway, I'll fix it.